Makefile

```diff
@@ -70,6 +70,11 @@ deps: $(GOPATH)/bin/godeps
 create-deps: $(GOPATH)/bin/godeps
 	godeps -t $(shell go list $(PROJECT)/...) > dependencies.tsv || true
 
+# Generate man pages.
+man: install
+	mkdir -p man
+	cd man && ../scripts/generate-all-manpages.sh
+
```

Comment:
The 'man' target is correctly not a .PHONY but it does depend on a .PHONY, so running 'make man' repeatedly rebuilds the man pages each time.  That's not very make-like.

If you removed the dependency on 'install' and instead had a 
   $(MAKE) install

action as the first thing in the man rule, then it would behave as expected.  Unless, of course, you want things to be regenerated all the time.

Also, you should remove the man directory as part of 'make clean'.